examples/query-action/query_action.metatags

```diff
@@ -0,0 +1,2 @@
+description: Example on using Ballerina query action.
```

Comment:
Ideally, the description should be between 50-160 characters. This is 40 characters.